sdk/keyvault/keyvault-common/.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "plugins": ["@azure/azure-sdk"],
+  "extends": ["plugin:@azure/azure-sdk/azure-sdk-base"],
+  "ignorePatterns": ["src/core"],
+  "rules": {
+    "@typescript-eslint/no-this-alias": "off",
+    "no-invalid-this": "off",
```

Comment:
is it necessary to turn those off here?

Reply:
I might take some days to address the feedback here, but it is appreciated!

Reply:
take your time!

Reply:
Done! Please review again! 🌞